reactos/drivers/usb/usbport/usbport.h

```diff
@@ -565,19 +565,20 @@ USBPORT_FindCompanionControllers(
   IN BOOLEAN IsObRefer,
   IN BOOLEAN IsFDOsReturned);
 
+VOID
+NTAPI
+USBPORT_InvalidateControllerHandler(
+  IN PDEVICE_OBJECT FdoDevice,
+  IN ULONG Type);
+
 /* debug.c */
 ULONG
 NTAPI
```

Comment:
__stdcall doesn't support varargs. I think the compilers usually warn about this?